### PR TITLE
getValues function added

### PR DESCRIPTION
**Pull request by javisantana**

### CODE REVIEW

File: lib/torque/renderer/point.js [hunk 1]
```diff
@@ -313,6 +313,28 @@ var Filters = require('./torque_filters');
       return positions;
     },
 
+    /**
+     * returns an array with all the values for the active pixels
+     * @tile tile object
+     * @step integer with the step
+     * @values (optional) an array where the values will be placed
+     */
+    getValues: function(tile, step, values) {
```
Comment (fdansv) [inline lib/torque/renderer/point.js]: @javisantana why not getTileValues? It's a bit confusing to have the same name both for active tiles and for the selected tile

Comment (javisantana) [inline lib/torque/renderer/point.js]: If we have getValueFor I think the name should be called getValues, if not the other one should be getTileValuesFor

Comment (fdansv) [inline lib/torque/renderer/point.js]: If you don't think there'll be confusion with the method naming, I've no objections.

**Comment by rochoa** [comment]
:es:

**Pull request merged**